javascript/javascript1/week1/homework_soheib/houseyPricey.js

```diff
@@ -0,0 +1,40 @@
+let peterHouse = {
+    width: 8,
+    depth: 10,
+    height: 10,
+    garden: 100,
+    price: 2500000
+};
+
+let juliaHouse = {
+    width: 5,
+    depth: 11,
+    height: 8,
+    garden: 70,
+    price: 1000000
+};
+
+function volumeCalc(house){
```

Comment:
didn't know you'd learned about functions yet! ;) Really nice approach

Reply:
Sorrryyy :D 
I didn't think it was a big cheat, because we learned about function in the preparation materials anyway, and the practice goal of the exercise (if and comparisons) is inside the function. so two birds one logic :D